### DISCUSSION TITLE
gnome can not find virt-manager 4.1 icon ,although I have installed successfully, Who can help me solve this problem? Thank you

### DISCUSSION THREAD

**Discussion by yuhmqq** [virt-manager/virt-manager#660] [General]
1-------------------------------------------------
[root@compute1 ~]# virt-manager
Traceback (most recent call last):
  File "/usr/bin/virt-manager", line 6, in <module>
    from virtManager import virtmanager
  File "/usr/local/virt-manger-4.1.0/share/virt-manager/virtManager/virtmanager.py", line 13, in <module>
    import gi
ModuleNotFoundError: No module named 'gi'
2------------------------------------------------------------------
 gnome can not find virt-manager 4.1 icon

**Comment by phrdina** [comment]
Hi, sorry for taking long time to reply. The main issue is not having `gi` python module. Try to see if your distribution has package for it, could be named differently, for example on Fedora it is `python3-gobject`.
